frontend/src/index.pug

```diff
@@ -34,6 +34,14 @@ html
     script(src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.14/build/languages/yaml.min.js")
   body
     #app
+      loading-overlay.overlay-loader(
+        v-bind:active.sync="isLoadingOverlayEnabled",
+        v-bind:opacity='loadingOverlayOpacity',
+        v-bind:is-full-page="true"
```

Comment:
Shouldn't v-cloak better to be added here?

Reply:
Meanwhile, the travis is failing.

Reply:
fixed